docs/yml.md

```diff
@@ -102,7 +102,7 @@ An entry with the ip address and hostname will be created in `/etc/hosts` inside
 ### ports
 
 Expose ports. Either specify both ports (`HOST:CONTAINER`), or just the container
```

Comment:
Looks good. The only thing I'd change is this line:

> Once a host is running, use the `docker port` command to see the actual mapping.

This should instead read `docker-compose port`.